### PR TITLE
Plate interpretation in Table.parseJson

### PR DESCRIPTION
**Pull request by djspiewak**
Had to move the plates out of mimir and into yggdrasil in order to keep `parseJson`'s implementation confined to that package. Ties the knot on everything mimir-related though.

### CODE REVIEW

**Comment by alissapajer** [comment]
@djspiewak it failed to compile.

**Comment by djspiewak** [comment]
Details…

File: yggdrasil/src/main/scala/quasar/yggdrasil/table/ColumnarTableModule.scala [hunk 1]
```diff
@@ -370,14 +371,30 @@ trait ColumnarTableModule
...
         precise: Boolean = false)(
         implicit ec: ExecutionContext)
         : M[Table] = {
 
       import fs2.{Chunk, Stream}
+      import tectonic.Plate
       import tectonic.json.Parser
 
-      val parser = Parser(new SlicePlate(precise), Parser.ValueStream)
+      val plate = instructions.foldRight(new SlicePlate(precise): Plate[List[Slice]]) {
```
Comment (alissapajer) [inline yggdrasil/src/main/scala/quasar/yggdrasil/table/ColumnarTableModule.scala]: @djspiewak realized while writing my impl (which is basically identical to this :) ) that you'll have to reverse the `List[Instruction]` if you want to `foldRight`. Or we could generate the instructions in the reverse order. Either way.

Comment (djspiewak) [reply yggdrasil/src/main/scala/quasar/yggdrasil/table/ColumnarTableModule.scala]: Actually I'm pretty sure `foldRight` without `reverse` is what I want. A chain of delegating plates is basically the same as a linked list. The first plate which applies to the input stream will be the one which corresponds to the first element of the list.

Comment (alissapajer) [reply yggdrasil/src/main/scala/quasar/yggdrasil/table/ColumnarTableModule.scala]: We're currently generating instructions in this order:
```
List(Mask, Wrap, Pivot)
```
Does this implementation evaluate them in the correct order (`Mask` first, followed by `Wrap`, then `Pivot`)?

Comment (djspiewak) [reply yggdrasil/src/main/scala/quasar/yggdrasil/table/ColumnarTableModule.scala]: It does. :-)

Comment (alissapajer) [reply yggdrasil/src/main/scala/quasar/yggdrasil/table/ColumnarTableModule.scala]: :+1: then we be good :)

**Pull request merged**